skfda/preprocessing/smoothing/__init__.py

```diff
@@ -1,2 +1,3 @@
 from . import kernel_smoothers
 from . import validation
+from ._basis import BasisSmoother
```

Comment:
I though about this, but I was not sure. The other smoothers are all kernel smoothers, so a submodule made sense, but in this case, it did not.

Reply:
Having only 4 smoothers, which receive an ``FDataGrid`` and return another ``FDataGrid`` by default, I don't know if a submodule is necessary. In my opinion, it is more important that related objects, with the same function and structure, have the same hierarchy in the api.

I would put them all in the ``smoothing`` module directly, because the functionality of the objects has more weight for me, but it is true that theoretically are two different types of smoothers. I guess it's a personal point of view.

Reply:
I will not change them for now, because they are related, and deserve an specific submodule. For now, I think that aliasing them directly in the `smoothing` module could be confusing, but I could reconsider that when more smoothers are added.